docs/06-objects-of-interest/02-config.md

```diff
@@ -679,6 +679,57 @@ Service-specific configs. These will override all other config files.
     <td><code>string[]</code></td>
     <td></td>
   </tr>
+  <tr>
+    <td><code>warningText</code><br/>
+      Customize warning and restriction text messages in UI
+    </td>
+    <td>true</td>
+    <td>
+      <code>object[]</code><br/>
+      object properties <ul>
+        <li><code>prefix</code></li> (optional)
+        <li><code>prefixNoRights</code></li> (optional)
+        <li><code>unusablePrefix</code></li> (optional)
+        <li><code>validityDescription</code></li> (optional)
+        <li><code>imagePreviewFlag</code></li> (optional)
+      </ul>
+    </td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>warningText.validityDescription</code><br/>
+      Override the validity description in ImageExtras
+    </td>
+    <td>true</td>
+    <td>
+      <code>object[]</code><br/>
```

Comment:
Am I right in thinking this is a single nested object, not an array of objects? Looks like the convention elsewhere is to have a single row per object property, e.g.

```
warningText.imagePreviewFlag.alertCopy
warningText.imagePreviewFlag.warningCopy
... etc
```